Casks/m/macgamestore.rb

```diff
@@ -1,20 +1,26 @@
 cask "macgamestore" do
-  version "4.3.4,6084"
-  sha256 "9f4b86ba8726de7c00fa129392b5b3cfdb0af7b682dfedfa1e0b00e08ea0cf11"
+  version "5.3.0,6127"
+  sha256 "3cc7c56c08a78c998ed86ad6b698df2be75045be54bfd3d7db61959ae4291622"
 
-  url "https://www.macgamestore.com/api_clientapp/clientupdates/public/core6/MacGameStore_#{version.csv.first}_#{version.csv.second}.tgz"
+  url "https://www.macgamestore.com/api_clientapp/clientupdates/public/core6/MacGameStore_#{version.csv.first}_#{version.csv.second}.zip"
   name "MacGameStore"
   desc "Buy, download, and play your games"
   homepage "https://www.macgamestore.com/app/"
 
   livecheck do
```

Comment:
```suggestion
  # This is a Sparkle feed but upstream uses a non-standard approach to
  # identify unstable versions that the `Sparkle` strategy doesn't handle, so
  #  we have to use the `Xml` strategy instead.
  livecheck do
```

We should explain why we're not using `Sparkle` for this appcast.